scripts/dgxctl.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 
 # Set configuration
-. configuration
+. config/containers/dgxie/configuration
 
 
 OPTIND=1
-IPMI_HOST_LIST="ipmi_host_list"
+IPMI_HOST_LIST="config/containers/dgxie/ipmi_host_list"
```

Comment:
This needs to be updated to config/pxe/ipmi_host_list and we should pull that list from containers/dgxie.